src/app/services/system-tray.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from '@angular/core';
+import { AppSettingsService } from './app-settings.service';
+import { debounceTime, filter, first, last, map, pairwise, take } from 'rxjs';
+import { invoke } from '@tauri-apps/api';
+
+const SYSTEM_TRAY_EXIT_COMMAND = 'set_exit_in_system_tray';
+const START_WITH_SYSTEM_TRAY_COMMAND = 'set_start_in_system_tray';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class SystemTrayService {
+  constructor(private readonly _appSettingsService: AppSettingsService) {
+  }
+  
+  async init() {
+    // Update exit in system tray behaviour following the setting.
+    this._appSettingsService.settings
+      .pipe(
+        map(settings => settings.exitInSystemTray),
+        pairwise(),
+        filter(([oldVal, newVal]) => oldVal !== newVal),
+        map(([_, newVal]) => newVal),
```

Comment:
```suggestion
        map(([, newVal]) => newVal),
```
Nit: in the case of unused array parameters, they can be left empty!